<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="L-INE-F7NSYZ2" failures="1" tests="1" name="com.scripts.EDD.TC14249_EnsureUserAlertedWhenCustomerWithanEDDIsInAwaitingApprovalVisits" time="66.215" errors="0" timestamp="18 Jun 2025 08:09:49 GMT" skipped="0">
  <testcase classname="com.scripts.EDD.TC14249_EnsureUserAlertedWhenCustomerWithanEDDIsInAwaitingApprovalVisits" name="tc14231_VerifyUserAlertedWhenCustomrWithEDDAwaitingApproval" time="66.215">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.generic.Utilities.addAssertTakeScreenShot(Utilities.java:383)
	at com.generic.Utilities.logReporter(Utilities.java:307)
	at com.pageFactory.desktopReception.CustomerEntry_UsingDOBName_PF.verifyEDDMessageDisplayWhileLogVisit(CustomerEntry_UsingDOBName_PF.java:91)
	at com.scripts.EDD.TC14249_EnsureUserAlertedWhenCustomerWithanEDDIsInAwaitingApprovalVisits.tc14231_VerifyUserAlertedWhenCustomrWithEDDAwaitingApproval(TC14249_EnsureUserAlertedWhenCustomerWithanEDDIsInAwaitingApprovalVisits.java:50)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- tc14231_VerifyUserAlertedWhenCustomrWithEDDAwaitingApproval -->
</testsuite> <!-- com.scripts.EDD.TC14249_EnsureUserAlertedWhenCustomerWithanEDDIsInAwaitingApprovalVisits -->
